### ISSUE TITLE
Uncaught TypeError: Cannot read property '174' of null

### ISSUE THREAD

**Issue by despairblue** [atom/atom#7678]
Was using undo and redo a lot.

**Atom Version**: 1.0.0
**System**: "Arch Linux"
**Thrown From**: Atom Core

### Stack Trace

Uncaught TypeError: Cannot read property '174' of null

```
At /usr/share/atom/resources/app.asar/node_modules/underscore-plus/node_modules/underscore/underscore.js:881

TypeError: Cannot read property '174' of null
  at /usr/share/atom/resources/app.asar/node_modules/underscore-plus/node_modules/underscore/underscore.js:881:18
  at Array.forEach (native)
  at _.each._.forEach (/usr/share/atom/resources/app.asar/node_modules/underscore-plus/node_modules/underscore/underscore.js:79:11)
  at Object._.defaults (/usr/share/atom/resources/app.asar/node_modules/underscore-plus/node_modules/underscore/underscore.js:878:5)
  at History.module.exports.History.popUndoStack (/usr/share/atom/resources/app.asar/node_modules/text-buffer/lib/history.js:94:11)
  at TextBuffer.module.exports.TextBuffer.undo (/usr/share/atom/resources/app.asar/node_modules/text-buffer/lib/text-buffer.js:709:30)
  at TextEditor.module.exports.TextEditor.undo (/usr/share/atom/resources/app.asar/src/text-editor.js:1142:19)
  at TextEditor.atom.commands.add.stopEventPropagation.core:undo (/usr/share/atom/resources/app.asar/src/text-editor-element.js:320:19)
  at atom-text-editor.newCommandListeners.(anonymous function) (/usr/share/atom/resources/app.asar/src/text-editor-element.js:288:32)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/usr/share/atom/resources/app.asar/src/command-registry.js:241:29)
  at /usr/share/atom/resources/app.asar/src/command-registry.js:3:61
  at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/usr/share/atom/resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:520:16)
  at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/usr/share/atom/resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:355:22)
  at HTMLDocument.module.exports.WindowEventHandler.onKeydown (/usr/share/atom/resources/app.asar/src/window-event-handler.js:180:20)

```

### Commands

```
  2x -1:20.2.0 core:backspace (atom-text-editor.editor.is-focused)
  2x -1:17.8.0 editor:newline (atom-text-editor.editor.is-focused)
  6x -1:10.4.0 core:backspace (atom-text-editor.editor.is-focused)
     -1:03.2.0 editor:delete-to-beginning-of-word (atom-text-editor.editor.is-focused)
     -0:22.3.0 core:backspace (atom-text-editor.editor.is-focused)
     -0:21.3.0 find-and-replace:select-next (atom-text-editor.editor.is-focused)
     -0:19.2.0 linter:set-bubble-transparent (atom-text-editor.editor.is-focused)
  3x -0:19 block-travel:move-down (atom-text-editor.editor.is-focused)
     -0:17.6.0 core:move-up (atom-text-editor.editor.is-focused)
     -0:16.9.0 editor:consolidate-selections (atom-text-editor.editor.is-focused)
     -0:16.4.0 core:move-up (atom-text-editor.editor.is-focused)
  2x -0:15.8.0 editor:newline (atom-text-editor.editor.is-focused)
     -0:14 core:paste (atom-text-editor.editor.is-focused)
     -0:12.8.0 core:cut (atom-text-editor.editor.is-focused)
  4x -0:12.8.0 core:undo (atom-text-editor.editor.is-focused)
     -0:08.9.0 core:redo (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {
    "excludeVcsIgnoredPaths": false,
    "disabledPackages": [
      "atom-color-highlight",
      "atom-pair",
      "atomatigit",
      "autocomplete",
      "autocomplete",
      "es-navigation",
      "filetype-color",
      "grunt-runner",
      "incremental-search",
      "linter-eslint",
      "linter-jshint",
      "linter-scspell",
      "markdown-scroll-sync",
      "metrics",
      "minimap-color-highlight",
      "minimap-find-and-replace",
      "minimap-git-diff",
      "minimap-highlight-selected",
      "minimap",
      "navigate",
      "package-cop",
      "starred-package-installer",
      "symbols-tree-view",
      "tab-smart-sort",
      "Tern",
      "test-status",
      "themed-settings",
      "travis-ci-status",
      "tree-view-breadcrumb",
      "tree-view-open-files",
      "welcome",
      "zentabs",
      "nuclide-click-to-symbol",
      "nuclide-quick-open"
    ],
    "projectHome": "/home/despairblue/vcs/git/",
    "destroyEmptyPanes": false,
    "autoHideMenuBar": true
  },
  "editor": {
    "fontFamily": "Source Code Pro, Symbola, Droid Sans Fallback",
    "showIndentGuide": true,
    "scrollPastEnd": true,
    "invisibles": {},
    "softWrap": true,
    "zoomFontWhenCtrlScrolling": false
  }
}
```

### Installed Packages

```coffee
# User
angularjs, v0.3.0
atom-alignment, v0.12.1
atom-beautify, v0.28.7
atom-fuzzy-grep, v0.4.0
atom-ternjs, v0.6.7
atom-zeal, v0.1.3
autocomplete-emojis, v2.2.2
autocomplete-paths, v1.0.2
block-travel, v1.0.2
bug-report, v0.7.1
color-picker, v2.0.7
compare-files, v0.6.1
editorconfig, v1.0.1
file-icons, v1.5.8
git-diff-popup, v0.1.8
git-log, v0.4.1
git-plus, v5.2.2
go-plus, v3.3.12
grammar-token-limit, v0.1.1
highlight-selected, v0.10.1
hyperclick, v0.0.0
indentation-indicator, v0.6.0
javascript-snippets, v1.0.0
jumpy, v2.0.9
language-dockerfile, v0.1.0
language-dot, v1.6.0
language-fish-shell, v1.0.2
language-haskell, v1.1.6
language-latex, v0.6.1
language-nimrod, v0.1.0
language-puppet, v0.12.0
language-tss, v0.4.0
latex, v0.24.2
linter, v1.1.0
linter-coffeelint, v0.3.2
linter-flow, v0.1.6
linter-htmlhint, v0.0.17
linter-js-standard, v1.0.5
linter-js-yaml, v1.1.1
linter-puppet-lint, v0.2.9
linter-shellcheck, v0.0.8
markdown-preview-opener, v0.1.1
nuclide-code-format, v0.0.0
nuclide-diff-view, v0.0.0
nuclide-file-tree, v0.0.0
nuclide-file-watcher, v0.0.0
nuclide-flow, v0.0.0
nuclide-hack, v0.0.0
nuclide-hg-repository, v0.0.0
nuclide-installer, v0.0.0
nuclide-language-hack, v0.0.0
nuclide-remote-projects, v0.0.0
nuclide-type-hint, v0.0.0
package-sync, v0.2.0
pane-layout-plus, v0.7.3
pdf-view, v0.22.0
pigments, v0.8.2
project-manager, v1.15.10
react, v0.12.3
regex-railroad-diagram, v0.8.1
swap-panes, v0.2.0
tabs-to-spaces, v0.11.0

# Dev
linter, v1.1.0
```

**Comment by maxbrunsfeld** [comment]
Related: https://github.com/atom/atom/issues/7352

**Comment by maxbrunsfeld** [comment]
I think this cannot happen anymore. The fix will be in 1.0.3.
